lib/active_merchant/connection.rb

```diff
@@ -82,6 +82,11 @@ def request(method, body, headers = {})
               # very unambiguously does not.
               raise ArgumentError, "DELETE requests do not support a request body" if body
               http.delete(endpoint.request_uri, headers)
+            when :delete_with_body
+              debug body
```

Comment:
This should probably be deleted.

Reply:
This is actually the standard for any request with a `body`

https://github.com/activemerchant/active_merchant/blob/3b428287e5f2d96d41c986abe0a83fb72032e2c9/lib/active_merchant/connection.rb#L77

Reply:
Neat, I see that now.